### ISSUE TITLE
Community Metrics: Share of Voice

### ISSUE THREAD

**Issue by GeorgLink** [chaoss/metrics#113]
I came across a metric that I consider worth sharing: Share of Voice

Source: https://medium.com/@mbbroberg/community-metrics-share-of-voice-b51d15f38366

This was not on my radar and I don't think we have it on our list of metics (haven't checked though).

**Comment by germonprez** [comment]
@GeorgLink should we simply include this in the giant list?

**Comment by GeorgLink** [comment]
Yes

**Comment by germonprez** [comment]
@GeorgLink Is this a D&I issue?

**Comment by GeorgLink** [comment]
Yes, I was made aware of this metric when I did my D&I metrics interviews.
